lib/facter/lvm_support.rb

```diff
@@ -4,8 +4,8 @@
   confine :kernel => :linux
 
   setcode do
-    vgdisplay = Facter::Util::Resolution.which('vgs')
-    vgdisplay.nil? ? nil : true
+    `which vgs`
+    $?.success? ? true : nil
```

Comment:
I noticed that the return value of the ternary operator are reverse of the original. Is this on purpose? [Ternary operators](http://alvinalexander.com/blog/post/ruby/examples-ruby-ternary-operator-true-false-syntax) are confusing sometimes.
This is what is causing one of the tests to fail.